deps/hdr_histogram/hdr_histogram.h

```diff
@@ -110,6 +110,18 @@ size_t hdr_get_memory_size(struct hdr_histogram* h);
  */
 bool hdr_record_value(struct hdr_histogram* h, int64_t value);
```

Comment:
Is this still being used anywhere? Can it be removed?

If not, and you're keeping it for compat purposes, should the method docs be updated to indicate it's deprecated or something?

Reply:
> Is this still being used anywhere? 

@DrEsteban it's coming from the original repo: https://github.com/HdrHistogram/HdrHistogram_c
I believe it's best to create an issue there and document this further on the function signature.

Reply:
Went ahead and did that for you: https://github.com/HdrHistogram/HdrHistogram_c/issues/126

Please add details or clarifications as necessary.